.github/workflows/test-old-branches.yml

```diff
@@ -59,7 +61,7 @@ jobs:
     steps:
       - name: Dispatch workflow run
         uses: actions/github-script@441359b1a30438de65712c2fbca0abe4816fa667 # v5.0.0
-        if: ${{ github.event_name == 'push' || github.event.schedule == '0 0 15 * *' || matrix.branch == '5.7' }}
+        if: ${{ github.event_name == 'push' || github.event.schedule == '0 0 15 * *' || matrix.branch == '5.9' }}
```

Comment:
This was 5.7. The example shows that it should have been 5.8 https://github.com/WordPress/wordpress-develop/pull/1199.